internal/ast/ast.go

```diff
@@ -166,9 +166,10 @@ type (
 	// ObjectExpr node represents a object creation expression.
 	ObjectExpr struct {
 		Position
-		Name *Ident // Name of identifier or nil.
-		Type Expr   // Type of object or nil.
-		Elts []Expr // Elements used to created object or nil.
+		Name    *Ident // Name of identifier or nil.
+		Type    Expr   // Type of object or nil.
+		Elts    []Expr // Elements used to created object or nil.
+		Comment string // Type of the object expr
```

Comment:
I think that would be good inform that this field don't have any semantic significance.
`// Optional type of the object expression; no semantic significance`